Documentation/gettingstarted/k3s.rst

```diff
@@ -23,6 +23,11 @@ Install a Master Node
 The first step is to install a K3s master node making sure to disable support
 for the default CNI plugin and the built-in network policy enforcer:
 
+.. note::
+
+   If Run Cilium in ``kube-proxy`` free mode, add ``--disable-kube-proxy``
+:ref:`Kube-proxy free <https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/>`_.
```

Comment:
You should be able to do ``` :ref:`kubeproxy-free` ``` to build the link automatically. Then the older versions of the docs will point to the correct current version of the doc instead of redirecting users to the latest version. It will also continue to work if we move that page elsewhere (eg as a configuration page instead of a GSG).

Reply:
yes, i was wondering in the same doc with :ref:`troubleshooting_k8s` that build the link automatically, not sure what the magic behind it, and how to use the correct string like :ref:`kubeproxy-free` to build the link automatically, or any string would do it?

Reply:
There's a magic line above the title in that other doc that looks like this:

```
$ git grep -A 4 kubeproxy-free:
Documentation/gettingstarted/kubeproxy-free.rst:.. _kubeproxy-free:
Documentation/gettingstarted/kubeproxy-free.rst-
Documentation/gettingstarted/kubeproxy-free.rst-*****************************
Documentation/gettingstarted/kubeproxy-free.rst-Kubernetes Without kube-proxy
Documentation/gettingstarted/kubeproxy-free.rst-*****************************
```

So then if you take the line between the `.. _` and the `:`, that's the link you can use in ``` :ref:`kubeproxy-free` ```.

Reply:
Also worth using `make render-docs` locally to browse to the page and see how it shows up.